examples/psp22-extension/runtime/psp22-extension-example.rs

```diff
@@ -0,0 +1,398 @@
+use codec::{
+    Decode,
+    Encode,
+    MaxEncodedLen,
+};
+use frame_support::{
+    dispatch::RawOrigin,
+    log::{
+        error,
+        trace,
+    },
+    pallet_prelude::*,
+    traits::fungibles::{
+        approvals::{
+            Inspect as AllowanceInspect,
+            Mutate as AllowanceMutate,
+        },
+        Inspect,
+        InspectMetadata,
+        Transfer,
+    },
+};
+use pallet_assets::{
+    self,
+    WeightInfo,
+};
+use pallet_contracts::chain_extension::{
+    ChainExtension,
+    Environment,
+    Ext,
+    InitState,
+    RetVal,
+    SysConfig,
+    UncheckedFrom,
+};
+use sp_runtime::{
+    traits::{
+        Saturating,
+        StaticLookup,
+        Zero,
+    },
+    DispatchError,
+};
+
+#[derive(Debug, PartialEq, Encode, Decode, MaxEncodedLen)]
+struct Psp22BalanceOfInput<AssetId, AccountId> {
+    pub asset_id: AssetId,
+    pub owner: AccountId,
```

Comment:
Don't think these fields need to be `pub` (same for all the other structs)

Reply:
This still hasn't been addressed

Reply:
Weird, I remember doing this. Maybe a commit got lost somehow. 😕